adcom1/audio_placement.go

```diff
@@ -194,7 +194,7 @@ type AudioPlacement struct {
 	//   video ad pod. If multiple impression opportunities within a bid request
 	//   share the same podid, this indicates that those impression opportunities
 	//   belong to the same video ad pod.
-	PodID int64 `json:"podid,omitempty"`
+	PodID string `json:"podid,omitempty"`
```

Comment:
The AdCOM PodIDs are integers, per: https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md

Looks like the ad podding changes to AdCOM (March 2022) happened before the OpenRTB 2.6 spec was finalized (October 2022), and now there's a mismatch. Let's keep the adcom package int64 to match the spec, but openrtb2 should use a string. I think I see where my original gaffe came from. :)

Reply:
Got it. Reverted that change.